recipe/meta.yaml

```diff
@@ -1,178 +1,334 @@
+      script_env:
+        RELEASE=1
     requirements:
       host:
+        - python
         - pip
-        - python <3.9
-        - wheel
         - setuptools
+        - wheel
       run:
-        - {{ pin_subpackage(name + '-core') }}
-        - {{ pin_subpackage(name + '-features') }}
-        - psutil >=5.7.3,<5.9
-        - networkx >=2.3,<3.0
+        - python
+        - numpy <1.29,>=1.21
+        - pandas <2.2.0,>=2.0.0
+        - scikit-learn <1.5,>=1.3.0
+        - {{ pin_subpackage(name + '.common', exact=True) }}
     test:
+      source_files:
+        - features/tests
+      imports:
+        - autogluon.features
       requires:
         - pip
         - pytest
-      imports:
-        - autogluon.tabular
       commands:
         - pip check
-        - cd tabular && pytest
+        - pytest -v features/tests -k 'not test_check_style'
+        - python -c 'import autogluon.features.version; assert autogluon.features.version.__version__ == "{{ version }}"'
 
-  - name: {{ name }}-mxnet
-    script: install_mxnet.sh
+  - name: {{ name }}.multimodal
+    build:
+      script: python -m pip install ./multimodal -v --no-deps --no-build-isolation
+      script_env:
+        RELEASE=1
     requirements:
       host:
+        - python
         - pip
-        - python <3.9
-        - wheel
         - setuptools
+        - wheel
       run:
-        - {{ pin_subpackage(name + '-core') }}
-        - mxnet
-        - pillow
+        - python
+        - numpy <1.29,>=1.21
+        - scipy <1.13,>=1.5.4
+        - pandas <2.2.0,>=2.0.0
+        - scikit-learn <1.5,>=1.3.0
+        - Pillow <11,>=10.0.1
+        - tqdm <5,>=4.38
+        - boto3 <2,>=1.10
+        - pytorch <2.1,>=2.0
+        - lightning <2.1,>=2.0.0
+        - requests <3,>=2.21
+        - jsonschema <4.18,>=4.14
+        - seqeval <1.3.0,>=1.2.2
+        - evaluate <0.5.0,>=0.4.0
+        - huggingface_accelerate <0.22.0,>=0.21.0
+
+        # transformers[sentencepiece]
+        - transformers <4.32.0,>=4.31.0
+        - sentencepiece !=0.1.92,>=0.1.91
+
+        - timm <0.10.0,>=0.9.5
+        - torchvision <0.16.0,>=0.14.0
+        - scikit-image <0.21.0,>=0.19.1
+        - text-unidecode <1.4,>=1.3
+        - torchmetrics <1.2.0,>=1.0.0
+        - nptyping <2.5.0,>=1.4.4
+        # Deviating from upstream. omegaconf-2.2.3 isn't fully compatible with Python 3.11.
+        - omegaconf <2.4.0,>=2.1.1
+        - {{ pin_subpackage(name + '.core', exact=True) }}
+        - {{ pin_subpackage(name + '.features', exact=True) }}
+        - {{ pin_subpackage(name + '.common', exact=True) }}
+        - pytorch-metric-learning <2.0,>=1.3.0
+        - nlpaug <1.2.0,>=1.1.10
+        - nltk <4.0.0,>=3.4.5
+        - openmim <0.4.0,>=0.3.7
+        - defusedxml <0.7.2,>=0.7.1
+        - jinja2 <3.2,>=3.0.3
+        - tensorboard <3,>=2.9
+        - pytesseract <0.3.11,>=0.3.9
+        # Exclusing because of imcompatibility with nvidia-ml-py.
```

Comment:
I'll clarify in the comment. `nvidia-ml-py` explicitly conflicts with `nvidia-ml` since they both provide the same files. `nvidia-ml` is deprecated and shouldn't be used.

Reply:
where is nvidia-ml-py coming from?

Reply:
It's coming from a transitive dep:
```
autogluon
  ray
    gpustat
      nvidia-ml-py
```